enclave/DA/dynamodb_test.ts

```diff
@@ -0,0 +1,22 @@
+const item = { id:"124",test: 1542,lis: ["dfd",12],bigint: BigInt(1000000000000000000000.000000000001),settest: new Set([123, 456, 789])};
```

Comment:
Could you add a TODO, that file is for reference will be deleted

Reply:
good find, I meant this to not be included in the PR at all